conda/gateways/disk/delete.py

```diff
@@ -102,7 +107,21 @@ def move_path_to_trash(path, preclean=True):
 def backoff_unlink(file_or_symlink_path, max_tries=MAX_TRIES):
     def _unlink(path):
         make_writable(path)
-        unlink(path)
+        try:
+            unlink(path)
+        except (IOError, OSError) as e:
+            if on_win and e.errno == 13:
+                try:
+                    win32file.MoveFileEx(file_or_symlink_path, None,
+                                         win32file.MOVEFILE_DELAY_UNTIL_REBOOT)
+                    log.info("Windows thinks file %s is in use.  We have scheduled it for "
+                             "removal at next reboot." % file_or_symlink_path)
+                except pywintypes.error:
+                    log.info("Windows thinks file %s is in use.  We can't schedule it for removal"
```

Comment:
Should this be log.info or log.debug? If it is not an admin prompt, would it fill the end user's shell with this message multiple times, in an exponential decay?

Reply:
The `delete_trash` function logs failures to `log.info` so I think this is alright.

Reply:
No, it won't show it multiple times.  The point of this PR is to avoid the exponential decay completely - abort after the first failure.  It will show it once per file that fails to be removed.